src/compat/posix/include/unistd.h

```diff
@@ -17,6 +17,8 @@
 #include <sys/stat.h>
 #include <sys/cdefs.h>
 
+#include <sys/uio.h>
```

Comment:
Oh, sorry, you are rigth.

Malloc issue is the critical that prevent merging PR
28 апр. 2016 г. 16:14 пользователь "Denis Chusovitin" <
notifications@github.com> написал:

> In src/compat/posix/include/unistd.h
> <https://github.com/embox/embox/pull/932#discussion_r61424162>:
>
> > @@ -17,6 +17,8 @@
> >  #include <sys/stat.h>
> >  #include <sys/cdefs.h>
> >
> > +#include <sys/uio.h>
>
> Probably, I misunderstood something, but this file is already in embox
> repository:
>
> https://github.com/embox/embox/blob/master/src/compat/posix/include/sys/uio.h
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/embox/embox/pull/932/files/e6f03d96cdf7e1e8d7a19585947c1a42c6b403ff#r61424162>
>